Makefile

```diff
@@ -1,11 +1,20 @@
-.PHONY: qa cs cfx phpstan tests build
+.PHONY: project install setup qa dev cs csf phpstan tests build
+
+############################################################
+# PROJECT ##################################################
+############################################################
+
+project: install setup
```

Comment:
I think this stage is not needed anymore as there is no `setup` stage. 👍

Reply:
There must be setup phase, for deployment. 

```
setup:
	mkdir -p temp log
	chmod +0777 temp log
```

Reply:
why, it isn't in another skeletons?